src/runtime/runtime.ml

```diff
@@ -683,10 +685,10 @@ let print_external ?max_level ~penv v ppf =
 (** In the future this routine will be type-driven. One consequence is that
     constructor tags will be printed by looking up their names in type
     definitions. *)
-let rec print_value ?max_level ~penv v ppf =
+let rec print_value ?max_level ~penv ~sgn v ppf =
```

Comment:
Here again, either we don't use `~sgn` at all, or we put it inside `~penv`. In this case, I think, we should put it inside `~penv` (because this is the runtime printing environment).